Разметка всех страниц

KateSolodchuk opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/2357681/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2357681-sedona-1/2/

KateSolodchuk committed:
Размечает навигацию в шапке

KateSolodchuk committed:
Добавляет логотип в шапку

KateSolodchuk committed:
Начинает размечать основную часть главной страницы

Намечены логотип, секция с преимуществами, секция с особенностями отдыха и слоганы между ними

KateSolodchuk committed:
Начинает размечать подвал

KateSolodchuk committed:
Исправляет ссылки и начинает размечать каталог

Добавлены адреса ссылок в соцсетях в подвале, логотипы в подвале и шапке исправлены на ссылки, размечены шапка и подвал каталога

KateSolodchuk committed:
Размечает каталог

Добавлена разметка каталога кроме фильтра стоимости

KateSolodchuk committed:
Исправляет неточности и завершает разметку каталога

Проверила на лишние элементы, поправила ошибки

KateSolodchuk committed:
Доделывает главную и каталог полностью

Director099 commented on index.html:
```diff
@@ -4,9 +4,135 @@
+        <h2 class="advantages-title">Седона — небольшой городок в Аризоне,
+          заслуживающий большего!</h2>
+        <p class="advantages-description">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+        <ul class="advantages-list">
+          <li class="advantages-item">
+            <h3 class="advantages-list-title">Настоящий городок</h3>
+            <p class="advantages-list-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+            <img class="advantages-img" src="#" alt="Домики в Седоне">
+          </li>
+          <li class="advantages-item">
+            <h3 class="advantages-list-title">Небольшая площадь</h3>
+            <p class="advantages-list-description">Все достопримечательности находятся очень близко</p>
+          </li>
+          <li class="advantages-item">
+            <h3 class="advantages-list-title">Красивая дорога</h3>
+            <p class="advantages-list-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+          </li>
+          <li class="advantages-item">
+            <h3 class="advantages-list-title">Мало туристов</h3>
+            <p class="advantages-list-description">Большинство едет в Гранд Каньон и толпится там</p>
+          </li>
+          <li class="advantages-item">
+            <h3 class="advantages-list-title">Там есть мост Дьявола</h3>
+            <p class="advantages-list-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+            <img class="advantages-img" src="#" alt="Мост Дьявола">
+          </li>
+        </ul>
```
Лучше таким образом разметить http://joxi.ru/a2XJ0oetpPRzDr

Как ты реализовала, НЕ ОШИБКА, но могут быть сложности в построении сетки, это возможно, но сложнее, если сделаешь как у меня на скрине то будет проще

Director099 commented on index.html:
```diff
@@ -4,9 +4,135 @@
+        <h2 class="visually-hidden">Социальные сети</h2>
+        <ul class="socials-list">
+          <li class="socials-item">
+            <h3 class="socials-title">Ссылка на Вконтакте</h3>
+            <a class="socials-link" href="https://vk.com/htmlacademy"><img class="socials-img" src="#" alt="Vkontakte"></a>
+          </li>
+          <li class="socials-item">
+            <h3 class="socials-title">Ссылка на Телеграм</h3>
+            <a class="socials-link" href="https://t.me/htmlacademy"><img class="socials-img" src="#" alt="Telegram"></a>
+          </li>
+          <li class="socials-item">
+            <h3 class="socials-title">Ссылка на Ютуб</h3>
```
лучше класс socials-title заменить на `visually-hidden` заголовка ведь нету и полощжить внутрь ссылки

Director099 commented on catalog.html:
```diff
@@ -0,0 +1,240 @@
+        <span class="product-price">От 4000 ₽</span>
+        <a class="product-card-button-details" href="#">Подробнее</a>
+        <button class="product-card-button-select">В избранное</button>
+        <button class="product-card-button-select-active">В избранном</button>
```
два раза писать не нужно, сделай как в макете, чтоб только в 1ной карточке было 
`<button class="product-card-button-select active">В избранном</button>`

обрати внимание я отделил название класса, тоесть active перекрасит кнопку, об этом чуть позже на css расскажут

KateSolodchuk committed:
Исправляет замечания

Исправлены блоки с преимуществами на главной странице, с соцсетями в подвале и кнопка "в избранное" в карточках гостиниц нас странице каталога

KateSolodchuk commented:
Замечания исправлены)
